protocol/requests/enable_installation_and_pair.go

```diff
@@ -6,6 +6,11 @@ import (
 
 var ErrEnableInstallationAndPairInvalidID = errors.New("enable installation and pair: invalid installation id")
 
+type InstallationIDProvider interface {
+	GetInstallationID() string
+	Validate() error
+}
+
```

Comment:
In Go interfaces are usually defined in place of usage, so in this case somewhere next to `SendPairInstallation`.
Because `InstallationIDProvider` doesn't make any sense in the `requests` dir